### DISCUSSION TITLE
求解区域是一个圆

### DISCUSSION THREAD

**Discussion by xiang-tao** [weihuayi/fealpy#69] [Q&A] [answered]
魏老师，我打算模仿你写的Poission方程代码求解我自己的一个方程，但是求解区域是一个半径为1的单位圆，如果使用三角形单元进行剖分，我不知道下面的代码该怎么改才会是对一个圆进行剖分！谢谢老师
```
domain = pde.domain()
mesh = mf.boxmesh2d(domain, nx=n, ny=n, meshtype='tri')
```

**Comment by wangdong19** [comment, marked as answer]
meshfactory中有一个可以生成单位圆上非结构三角形的方法unitcirclemesh()，可以使用这个，代码如下:
mesh = mf.unitcirclemesh(0.1,meshtype='tri')
